addons/diagnostic/fnc_projectileTracking_trackProjectile.sqf

```diff
@@ -39,13 +43,9 @@ if (!isNull _projectile) then {
     } else {
         _bulletData = [_initialProjectileData];
         _data = [_handle, _bulletData];
+        GVAR(projectileData) set [_index, _data];
     };
-
-    _bulletData pushBack [(getPos _projectile), vectorMagnitude (velocity _projectile)];
-
-    _data set [1, _bulletData];
```

Comment:
did you delete this line intentional?

Reply:
```
{
    _bulletData = _data select 1;
} else {
    _data = [_handle, _bulletData];
};

_data set [1, _bulletData];
```
either way `_bulletData` is going to already be set in `_data`